### ISSUE TITLE
Use `alpine` variant of `eclipse-temurin` for base images with JDK

### ISSUE THREAD

**Issue by petertrr** [saveourtool/save-cloud#1031]
Docker images of `eclispe-temurin` have two variants: Ubuntu-based (e.g. `17`) and Alpine-based (e.g. `17-alpine`). Switching to Alpine can spare us some time on downloads, because its about 100MB smaller.

**Comment by petertrr** [comment]
After #1030

**Comment by petertrr** [comment]
We are currently building executables of save-agent and save-cli linked with glibc on Ubuntu. So they are unable to start on alpine.

```
/home/save-agent/save-execution # ls
agent.properties          save-0.3.2-linuxX64.kexe  save-agent.kexe           test-suites
/home/save-agent/save-execution # ./save-agent.kexe
sh: ./save-agent.kexe: not found
/home/save-agent/save-execution # ldd save-agent.kexe
        /lib64/ld-linux-x86-64.so.2 (0x7f86dd9c4000)
Error loading shared library libresolv.so.2: No such file or directory (needed by save-agent.kexe)
        libm.so.6 => /lib64/ld-linux-x86-64.so.2 (0x7f86dd9c4000)
        libpthread.so.0 => /lib64/ld-linux-x86-64.so.2 (0x7f86dd9c4000)
        libutil.so.1 => /lib64/ld-linux-x86-64.so.2 (0x7f86dd9c4000)
Error loading shared library libcrypt.so.1: No such file or directory (needed by save-agent.kexe)
        librt.so.1 => /lib64/ld-linux-x86-64.so.2 (0x7f86dd9c4000)
        libcurl.so.4 => /usr/lib/libcurl.so.4 (0x7f86dd947000)
        libdl.so.2 => /lib64/ld-linux-x86-64.so.2 (0x7f86dd9c4000)
Error loading shared library libgcc_s.so.1: No such file or directory (needed by save-agent.kexe)
        libc.so.6 => /lib64/ld-linux-x86-64.so.2 (0x7f86dd9c4000)
Error loading shared library ld-linux-x86-64.so.2: No such file or directory (needed by save-agent.kexe)
        libnghttp2.so.14 => /usr/lib/libnghttp2.so.14 (0x7f86dd922000)
        libssl.so.1.1 => /lib/libssl.so.1.1 (0x7f86dd8a1000)
        libcrypto.so.1.1 => /lib/libcrypto.so.1.1 (0x7f86dd61f000)
        libbrotlidec.so.1 => /usr/lib/libbrotlidec.so.1 (0x7f86dd613000)
        libz.so.1 => /lib/libz.so.1 (0x7f86dd5f9000)
        libbrotlicommon.so.1 => /usr/lib/libbrotlicommon.so.1 (0x7f86dd5d6000)
Error relocating save-agent.kexe: _Unwind_Resume: symbol not found
Error relocating save-agent.kexe: isnan: symbol not found
Error relocating save-agent.kexe: _Unwind_Backtrace: symbol not found
Error relocating save-agent.kexe: _Unwind_GetIP: symbol not found
Error relocating save-agent.kexe: _Unwind_GetDataRelBase: symbol not found
Error relocating save-agent.kexe: _Unwind_GetTextRelBase: symbol not found
Error relocating save-agent.kexe: _Unwind_GetRegionStart: symbol not found
Error relocating save-agent.kexe: _Unwind_GetLanguageSpecificData: symbol not found
Error relocating save-agent.kexe: _Unwind_GetIPInfo: symbol not found
Error relocating save-agent.kexe: _Unwind_SetGR: symbol not found
Error relocating save-agent.kexe: _Unwind_SetIP: symbol not found
Error relocating save-agent.kexe: _Unwind_RaiseException: symbol not found
Error relocating save-agent.kexe: _Unwind_DeleteException: symbol not found
Error relocating save-agent.kexe: _Unwind_Resume_or_Rethrow: symbol not found
Error relocating save-agent.kexe: __strftime_l: symbol not found
```

**Comment by akuleshov7** [comment]
@0x6675636b796f75676974687562 if you wish - it will be great if you will help Petr with this task

**Comment by petertrr** [comment]
I think this task can have a pretty low priority. Considering https://youtrack.jetbrains.com/issue/KT-38876 and https://youtrack.jetbrains.com/issue/KT-38891, this task can consume more time than return value.

**Comment by akuleshov7** [comment]
> I think this task can have a pretty low priority. Considering https://youtrack.jetbrains.com/issue/KT-38876 and https://youtrack.jetbrains.com/issue/KT-38891, this task can consume more time than return value.

true, as always in our team :D
